fixtures/keywords/README.md

```diff
@@ -0,0 +1,5 @@
+These fixtures are designed to test that keywords from supported languages
+are able to be used.
```

Comment:
Good question :) My first attempt at this PR did exactly that, but I found it unwieldy.

The problem with having one fixture that tried to use the union of keywords across languages demonstrated that it was too easy to miss coverage for a language.

ie, each new fixture tries to make sure the language keyword is used in every possible context - so we want a `dictionary`, `interface`, `enum`, `error` etc with keywords, plus where possible, each element, function/arg name, etc within those objects to use keywords.

In practice, this means we would need as many (say) `enums` as necessary to ensure all language keywords were used in an enum definition, and every named variant also used keywords etc.

The end result is that it would be extremely easy to accidentally discover that, say, we didn't have a `dictionary` which was a keyword name in one of the languages. Or in one of the arg names in the enum. Or in the function name in an interface, etc.

So while this means there are more fixtures, all of which are superficially similar, it's much easier to reason about the fact that the kotlin tests do actually use kotlin keywords in every possible context. Combining python+rust into a single fixture was an attempt to make this slightly less painful, but it actually made me re-look and discover that it's using `class`, which is not a Rust keyword, so coverage is actually missing for an enum variant being a rust keyword (which I will fix as part of this PR)! Not to mention custom/external types, which aren't part of this PR (and I think can land without them for now)

Let me know if that makes sense and is the correct trade-off - if so, I'll update the new README.md with this - otherwise, I'll have another poke at a single fixture covering all cases (there's really no urgency to land this, so I'm fine either way)

Reply:
That makes total sense to me.